src/project-tests/scatter-plot-tests.js

```diff
@@ -67,10 +67,38 @@ export default function createScatterPlotTests() {
                 const sortedDots = dots.sort(function(a, b) {
                     return a.getAttribute("data-xvalue") - b.getAttribute("data-xvalue")
                 });
-
-                //check to see if the x locations of the new sorted array are in ascending order
-                for (var i = 0; i < sortedDots.length - 1; ++i) {
-                    FCC_Global.assert.isAtMost(+sortedDots[i].cx.baseVal.value, +sortedDots[i + 1].cx.baseVal.value, "x values don't line up with x locations ");
+                var xAxisQuery = document.querySelector('#x-axis path');
```

Comment:
If you put this whole section into a new function (something like `checkDotsAxisAlign`?) I think it would make the tests more readable and focused on what is being tested. Also, I would like to see more documentation because I cannot figure out exactly what the code is doing just by reading the code. A summary at the top of the new function, and then some line comments where it's not obvious what the code is doing.